### ISSUE TITLE
/bin/sh: cross-env: command not found

### ISSUE THREAD

**Issue by clexi** [team-innovation/vue-sfc-rollup#21]
new installtion and get this error on `yarn build`

```
/bin/sh: cross-env: command not found
```

**Comment by mgdodge** [comment]
Single component mode, or library mode? Did you run npm install in the folder generated by the cli? More details, please. I just ran a test and didn't see any error...

**Comment by clexi** [comment]
Sorry Single Component mode. 

>  Did you run npm install in the folder generated by the cli? 

but this was my error... i'm sorry. 

But another question:
I also get an error in my nuxt.js project after importing my component.
```
ReferenceError
document is not defined
```
in `node_modules/my-component/dist/my-component.umd.js`. I thinks this error appears of SSR, should i miss something in my component configuration. I just tried the default generated component.

**Comment by mgdodge** [comment]
Inside the dist folder, there are 3 versions produced - umd, esm, and min. These are for use in bundlers and the browser. Which one is used should be resolved automatically when you import/require the file - ie. `require('my-component')` or use a script tag to load it from [unpkg](https://unpkg.com). For SSR scenarios, the original .vue file is also included in the package, and is aliased as `my-component/sfc`. So in the nuxt scenario, you could use `require('my-component/sfc')` to get the raw .vue file instead of the transpiled one.

I am going to close this issue since the original question was different. Hope this helps!

**Comment by clexi** [comment]
Yes Thanks a lot! 
i try to import the component via `my-component/sfc` but than nuxt.js can not find it. I can only use it on client side. I added it as a plugin into the nuxt.js project and only add this to ssr but it still can't be found. :(

**Comment by mgdodge** [comment]
It seems that nuxt/webpack is not resolving the `browser` field of package.json - seems a lot of people are complaining, acutally. Looking at the way nuxt configures webpack, it seems they might actually be actively avoiding it, TBH. Well, you can always use `require('my-component/src/my-component.vue)` in nuxt, and it will definitely work. That is what the alias would resolve to anyway, but I'd hoped that the `browser` field alias would have worked. The spec indicates it should.

**Comment by mgdodge** [comment]
@clexi Just published v2.0.0 of vue-sfc-rollup, which has been updated with some changes to address nuxt/quasar/SSR in general. Go ahead and check it out! Thanks for your report, it helped me find a couple longstanding deficiencies in this utility, and I am pretty pleased where it has ended up!
